package.json

```diff
@@ -206,6 +206,7 @@
     "@types/chai": "^4.2.11",
     "@types/fs-extra": "^9.0.1",
     "@types/jest": "25.1.4",
+    "@types/lodash": "^4.14.172",
```

Comment:
We already have an existing version of lodash in package.json

Reply:
This are just type definitions for lodash. Should I still remove this?

Reply:
No, you leave it.

Reply:
done